### ISSUE TITLE
Definitions missing in vcf header

### ISSUE THREAD

**Issue by wdecoster** [nanoporetech/medaka#58]
Hi,

bcftools complains about missing definitions in the vcf header:

```
[W::vcf_parse] INFO 'pos2' is not defined in the header, assuming Type=String
[W::vcf_parse] INFO 'q2' is not defined in the header, assuming Type=String
[W::vcf_parse_format] FORMAT 'GT' is not defined in the header, assuming Type=String
[W::vcf_parse_format] FORMAT 'GQ' is not defined in the header, assuming Type=String
[W::vcf_parse] INFO 'pos1' is not defined in the header, assuming Type=String
[W::vcf_parse] INFO 'q1' is not defined in the header, assuming Type=String
[E::bcf_write] Unchecked error (2), exiting
```

I'll write a script to modify 3000 vcf files before being able to merge the results, so not an urgent problem, but might be good for the next release.

Thanks!

**Comment by cjw85** [comment]
Hi Wouter,

Is this while using the v0.7.0-alpha pre-release? I believe the issue has been resolved already (commit [b4850f00](https://github.com/nanoporetech/medaka/commit/b4850f008e6c139fe1bc9f46fffc06b6accce9ff#diff-f824d1b5e7a78f78fc09909c0a1119f4R621)).

**Comment by wdecoster** [comment]
Aha! Totally looks like it is fixed indeed. Apologies, my data hasn't caught up with the update yet!
